### PR TITLE
chore(load-tests): Tweak PreSigned URL script and add improvements to the dashboard

### PR DESCRIPTION
**Pull request by themarcelor**
Tweak PreSigned URL script and add improvements to Grafana's dashboard:
-> Fix Errors Per Second graph
-> Add new Success Rate graph

Here's how it looks like:
image

image

### CODE REVIEW

File: load-testing/fence/presigned-url.js [hunk 1]
```diff
@@ -1,30 +1,64 @@
...
-        { duration: "15s", target: 50 },
-        { duration: "30s", target: 50 },
-        { duration: "15s", target: 0 },
-    ],
-    thresholds: {
-        http_req_duration: ["avg<100", "p(95)<200"]
-    },
-    noConnectionReuse: true,
+// __ENV.GUIDS_LIST is only set when `random-guids` is triggered
+const guids = GUIDS_LIST ? GUIDS_LIST.split(',') : '00037250-f2e5-47e2-863c-5f225c7f79e8';
```
[the remark this answers is not in the document]
Comment (themarcelor) [reply load-testing/fence/presigned-url.js]: Good catch.
I will push another commit.
Thanks!

File: load-testing/load-test-descriptor.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    { "duration": "1s", "target": 46 },
+    { "duration": "15s", "target": 46 },
+    { "duration": "1s", "target": 47 },
+    { "duration": "15s", "target": 47 },
+    { "duration": "1s", "target": 48 },
+    { "duration": "15s", "target": 48 },
+    { "duration": "1s", "target": 49 },
+    { "duration": "15s", "target": 49 },
+    { "duration": "1s", "target": 50 },
+    { "duration": "120s", "target": 50 }
```
Comment (atharvar28) [inline load-testing/load-test-descriptor.json]: we should come up with better scenarios to perform load testing! Load on a particular endpoint can be gradually increased with time and RPS to get bet out of the services. maybe some spike testing or stress testing!

File: load-testing/load-test-stress.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     { "duration": "60s", "target": 40 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 60 },
+     { "duration": "60s", "target": 70 },
+     { "duration": "60s", "target": 50 },
+     { "duration": "60s", "target": 30 },
+     { "duration": "60s", "target": 0 }
```
Comment (atharvar28) [inline load-testing/load-test-stress.json]: similarly here. A good scenario to get best out of the service!

File: load-testing/load-test-descriptor.json (hunk 2 above)
Comment (themarcelor) [reply load-testing/load-test-descriptor.json]: Feel free to add more sample scenarios in a new PR

File: load-testing/load-test-stress.json (hunk 3 above)
Comment (themarcelor) [reply load-testing/load-test-stress.json]: We can improve the documentation on where we should store sample scenarios.

**Pull request merged**